static/app/components/compactSelect/listBox/index.tsx

```diff
@@ -30,10 +31,6 @@ interface ListBoxProps
    * further callback function should be run.
    */
   keyDownHandler: (e: React.KeyboardEvent<HTMLUListElement>) => boolean;
-  /**
-   * Items to be rendered inside this list box.
-   */
-  listItems: Node<any>[];
```

Comment:
Filtering is now performed directly inside `ListBox`, rather than one level above in its parent, so we don't need to use this prop anymore.